packages/core/src/http-client/http-client.ts

```diff
@@ -201,11 +202,8 @@ function executeWithAxios(request: HttpRequest): Promise<HttpResponse> {
   return axios.request(mergeRequestWithAxiosDefaults(request));
 }
 
-function executeWithAxiosReturnRequestAndResponse(request: HttpRequest): Promise<HttpRequestAndResponse> {
-  const merged = mergeRequestWithAxiosDefaults(request);
-  return axios.request(merged).then(
-    res => ({ request: merged, response: res })
-  );
+function executeWithAxiosReturnDefaultResponse(request: HttpRequest): Promise<DefaultHttpResponse> {
```

Comment:
Now this function should not be needed anymore. We should change `executeWithAxios` to `function executeWithAxios(request: HttpRequest): Promise<HttpResponse>`

Reply:
you mean change to this `function executeWithAxios(request: HttpRequest): Promise<DefaultHttpResponse>`?